Makefile

```diff
@@ -1,5 +1,5 @@
 SKIP_SQUASH?=0
-VERSIONS="1"
+VERSIONS="1 slave-base slave-maven slave-nodejs"
```

Comment:
ok .. that helps, and yep, the pros/cons are such that there isn't a slam dunk choice for this.

How about adding a comment that lays out the rationals and implications you just detailed here for future reference?